lib/src/clixon_file.c

```diff
@@ -50,7 +50,8 @@
 #include <sys/param.h>
 #include <unistd.h>
 #include <netinet/in.h>
- 
+#include <stddef.h>
```

Comment:
What is the use of stddef.h in this context?

Reply:
`stddef.h` includes the `offsetof` macro that is used to calculate the size of the `dirent` struct.